js/content-slider.js

```diff
@@ -1,85 +1,81 @@
 export default class ContentSlider {
   /**
-   * TODO: 
-   * !! - Fix the Controls repositioning logic in the resizeObserver. 
+   * TODO:
+   * !! - Fix the Controls repositioning logic in the resizeObserver.
    * !! - Investigate the reason the controls click area diverges from img div on different browser zoom levels
    *  - Fix tab indexes (user should not be able to scroll by tabbing // add visibility classes )
-   *  - Refactor addition of event listeners into a separate function. DRY-er. 
-   * 
+   *  - Refactor addition of event listeners into a separate function. DRY-er.
+   *
    * Notes:
    *  - Clickable area off by 20px due to 2rem padding on .demo-section
    *  - Viewport top position start at [0, 79], not sure why.
    */
-  constructor(sliderWrapper = '.content-slider') {
-    // check if there is a content slider
-        this.contentSlider = document.querySelector(sliderWrapper);
+  constructor(
+    sliderWrapper = ".content-slider",
+    controlSurface = ".content-block__upper"
+  ) {
+  
+    this.contentSlider = document.querySelector(sliderWrapper);
+    this.controlSurface = document.querySelector(controlSurface);
 
     if (this.contentSlider) {
       /**
-       * TODO: update selectors to data-attributes
+       * TODO: Refactor controlSurface so that it appends the controls div to the DOM, therefore removing the necesity for a controlSurface parameter to be passed in.
```

Comment:
This was something I was just thinking about. I made a similar point on the lead form JS about success/error messages, and specifically removed the need to have markup in the page for the submitting overlay. This sort of change would help it degrade gracefully, but obviously can wait.